app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
@monfresh You are right that we don't need it in two places, now that I know we want to change the order in `SessionsController` (seemed fishy that the logic was backwards there).


right now, this partial is the only place we are using `lockout_time_remaiining_in_words` -- given that we are only using this method in a partial where there is no `current_user`, do you still think it belongs in the user decorator? (the answer might be yes, just wondering)